agents/demo/demo.ipynb

```diff
@@ -0,0 +1,970 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Agents on Kubeflow 🤓"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial we will be training a reinforcement learning agent from the [tensorflow/agents](https://github.com/tensorflow/agents) project on Kubernetes using [Kubeflow](https://github.com/google/kubeflow)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The task the agent will be learning to perform is to operate a Kuka Robotics arm simulated in the OpenAI Gym Bullet Physics 'KukaBulletEnv-v0' environment. Feel free to [skip to the end](http://localhost:8888/notebooks/kubeflow-rl/apps/agents_ppo/demo.ipynb#Rendering-the-model) to see what this will look like!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setup"
```

Comment:
I would prefer for the entire `Setup` section to be moved out to the README as opposed to put into the notebook.

I can see the value of having it in the notebook, whereby you can run the commands directly without having to switch back and forth between your browser and your terminal. However, these cells don't bring much value when running the notebook in kubeflow-core.

In the README, I would put one set of instructions (which is basically these) under a "Control locally" section and another set of instructions (as of now only implicitly provided in the demo notebook) under "Control form JupyterLab on kubeflow".